site/src/pages/TemplateSettingsPage/TemplateSchedulePage/TemplateScheduleForm/TemplateScheduleForm.tsx

```diff
@@ -158,6 +167,25 @@ export const TemplateScheduleForm: FC<TemplateScheduleForm> = ({
     }
   }
 
+  const handleToggleLockedCleanup = async (e: ChangeEvent) => {
```

Comment:
NIT: Since we now have 3 toggling functions, we might be able to refactor into one function that takes two form values, e.g. `locked_cleanup_enabled` and `locked_ttl_ms` and appropriately toggles TTL.